jujugui/static/gui/src/app/assets/css/_header.scss

```diff
@@ -0,0 +1,101 @@
+
+/*
+ * The main header above the canvas
+ */
+
+ %list-style {
+     margin: 0;
+     padding: 0;
+     list-style: none;
+ }
+
+.header-banner {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: $navbar-height;
+
+    .header-banner__link,
+    .header-banner__link--breadcrumb {
+        @extend .border-box;
+        color: #333;
```

Comment:
You could use `$text-colour` here as I've updated it to be `#333` in another branch (not yet landed, you could update `$text-colour` in base.scss to be `#333` in this branch too if you wanted).